storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/PersonGetResolver.java

```diff
@@ -0,0 +1,39 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Car;
+import com.pushtorefresh.storio.sample.db.entities.Person;
+import com.pushtorefresh.storio.sample.db.tables.CarsTable;
+import com.pushtorefresh.storio.sample.db.tables.UsersTable;
+import com.pushtorefresh.storio.sqlite.operations.get.DefaultGetResolver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PersonGetResolver extends DefaultGetResolver<Person> {
+
+    // We expect that cursor will contain both Tweet and User: SQL JOIN
+    @NonNull
+    @Override
+    public Person mapFromCursor(@NonNull Cursor cursor) {
+        final Person person = Person.newPerson(
+                cursor.getLong(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_ID)),
+                cursor.getString(cursor.getColumnIndexOrThrow(UsersTable.COLUMN_NICK))
+        );
+
+        List<Car> cars = new ArrayList<Car>();
+        while (cursor.moveToNext()) {
```

Comment:
Sure :)

As I see: one `person` may have multiple `cars` so in general you have 2 ways of retrieving Person + cars from the db:

1. Make query with `join` which will retrieve rows (`person`, `car`), but the problem is that if one `person` has multiple `cars` you will have duplicates of this `person` in next `n` rows, like this: `row1` ==  `person1`, `car1`, `row2` == `person1`, `car2`, `row3` == `person2`, `car3` and so on. It's possible and fast (for the db) way to retrieve `persons` and their `cars`.
2. Make query to only `persons` table and then attach `cars` for each `person` programmatically, but it will require a lot of queries to the `cars` table, which makes this solution slower than 1rst one

Now you need to decide which way you want to go :)